front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx

```diff
@@ -239,6 +252,7 @@ export const RollingStockEditorParameterForm = ({
   rollingStockValues,
   setOptionValue,
   setRollingStockValues,
+  currentRsEffortCurve,
```

Comment:
Do you think it's better to have currentRsEffortCurve or a boolean isElectric ? Maybe isElectric is better ?

Reply:
I need the my system is generic, if we ever need to add other conditions